### ISSUE TITLE
[Suggest] - add qbx_radialmenu

### ISSUE THREAD

**Issue by Mesrine67** [Qbox-project/qbx_smallresources#109]
### The problem

since qbx_radialmenu no longer uses its own ui interface but that of ox_lib, its code could be added to qbx_smallresources for better usability. 

### Ideal solution

example other resources can be added to qbx_smallresources like: qbx_fireworks, qbx_seatbelt

or another example add qbx_vehicles to qbx_core

**Comment by Manason** [comment]
I'm not sure radial menu needs it's own resource at all? It would work like ox_target, added to by other resources.

**Comment by Mesrine67** [comment]
yes, you're right maybe rename qbx_radiamenu to `qbx_interact`, this could create a resource that completes the game interaction using `Overextended`?

**Comment by Manason** [comment]
I think qbx_radialmenu is deprecated, with a long term goal to move everything into other resources and then delete it.

**Comment by Mesrine67** [comment]
Good to know
